crates/core/tedge_actors/src/builders.rs

```diff
@@ -113,25 +114,6 @@ pub trait ServiceConsumer<Request: Message, Response: Message, Config> {
     }
 }
 
-impl<T, Req, Res> ServiceConsumer<Req, Res, NoConfig> for T
-where
-    Req: Message,
-    Res: Message,
-    T: MessageSink<Res> + MessageSource<Req, NoConfig>,
-{
-    fn get_config(&self) -> NoConfig {
-        NoConfig
-    }
-
-    fn set_request_sender(&mut self, request_sender: DynSender<Req>) {
-        self.register_peer(self.get_config(), request_sender)
-    }
-
-    fn get_response_sender(&self) -> DynSender<Res> {
-        self.get_sender()
-    }
-}
```

Comment:
I removed this generic implementation because *too* generic and preventing specializations due to Rust complaining of possible future conflicting implementations.